### DISCUSSION TITLE
What is the Application Default Credentials story for alpha - no `credential` in ESM

### DISCUSSION THREAD

**Discussion by akauppi** [firebase/firebase-admin-node#1349] [General]
https://medium.com/google-cloud/firebase-separating-configuration-from-code-in-admin-sdk-d2bcd2e87de6

I found about ADC last week, and would like to try it out. However, `firebase-admin` 9.100.0-alpha.1 `exports` section does not have `credentials`. 

What is the way to do the same with "alpha"?  Is this going away - or the `credentials` just hasn't made it to this release, yet?

---

It turns out I won't need this (since the context is using Admin SDK within Cloud Functions). This may be the reason for the current omission - one cannot use "alpha" within Cloud Functions, anyways.

**Comment by hiranya911** [comment]
All functions in the `credentials` namespace are also exposed from the `firebase-admin/app` entry point. For example:

https://modular-admin.web.app/reference/firebase-admin.app#applicationdefault

But typically, when you use ADC in environments like Cloud Functions you don't need them at all. You can just call `initializeApp()` without any arguments.
